test/test_framework.py

```diff
@@ -1517,9 +1524,19 @@ def test_named_indicated_specifically(self, fake_stderr):
         # Some breakpoint was indicated, and the framework call used exactly that name.
         self.check_trace_set('mybreak', 'mybreak', 1)
 
-    def test_named_indicated_somethingelse(self, fake_stderr):
-        # Some breakpoint was indicated, but the framework call was not with that name.
+    def test_named_indicated_unnamed(self, fake_stderr):
+        # Some breakpoint was indicated, but the framework call was unnamed
         self.check_trace_set('some-breakpoint', None, 0)
+        self.assertLoggedWarning(
+            "Breakpoint None skipped",
+            "not found in the indicated ones: ['some-breakpoint']")
```

Comment:
what about "not found in the requested breakpoints: ['some-breakpoint']" ?

Reply:
Wording improved, thanks!